.travis.yml

```diff
@@ -61,6 +61,19 @@ matrix:
     - env: CLANG_VERSION=3.8 BUILD_TYPE=Release
       os: linux
       addons: *clang38
+    - env: CLANG_VERSION=3.9 BUILD_TYPE=Debug
```

Comment:
We should replace the occurrences of "3.9.0" on lines 13 and 126 with "3.9.1" to get the more recent libc++.